docs/DeveloperGuide.md

```diff
@@ -313,7 +313,35 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 2.
 
-*{More to be added}*
+**Use Case: Scheduling a meeting with a new person**
+
+**MSS**
+
+1. User adds new person to FINDvisor
+2. User requests to list persons
+3. FINDvisor shows a list of persons
+4. User requests to schedule a meeting with a specific person in the lsit
+5. Meeting is scheduled
+
```

Comment:
```suggestion
    Use case ends.
```
I think we should indicate that use case ends here for MSS (following the example given in AB3 guide)?